packages/services/src/kernel/messages.ts

```diff
@@ -152,6 +153,26 @@ namespace KernelMessage {
     return msg.header.msg_type === 'display_data';
   }
 
+  /**
+   * An `'update_display_data'` message on the `'iopub'` channel.
+   *
+   * See [Update Display data](https://jupyter-client.readthedocs.io/en/latest/messaging.html#update-display-data).
+   */
+  export
+  interface IUpdateDisplayDataMsg extends IDisplayDataMsg {
```

Comment:
Is this something that can be simplified with conditional types?

Reply:
I suppose we could use a MakeRequired typing thing, but I think it's simpler to be more explicit here.